libraries/libxml/source/org/pentaho/reporting/libraries/xmlns/writer/XmlWriterSupport.java

```diff
@@ -794,14 +803,19 @@ public void writeTextNormalized( final Writer writer, final String s, final bool
    * @param transformNewLine true, if a newline in the string should be converted into a character entity.
    * @return the normalised string.
    */
-  public String normalize( final String s, final boolean transformNewLine ) throws IOException {
+  public static String normalize( final String s, final boolean transformNewLine ) {
```

Comment:
It's a small urgent fix for previous PR. https://github.com/pentaho/pentaho-reporting/pull/889. The method "normalize" already was static.